raphtory-graphql/src/model/graph/graphs.rs

```diff
@@ -0,0 +1,48 @@
+use crate::model::graph::property::GqlProperties;
+use dynamic_graphql::{ResolvedObject, ResolvedObjectFields};
+use itertools::Itertools;
+use raphtory::{
+    db::api::{properties::dyn_props::DynProperties, view::DynamicGraph},
+    prelude::GraphViewOps,
+    search::{into_indexed::DynamicIndexedGraph, IndexedGraph},
+};
+
+#[derive(ResolvedObject)]
+pub(crate) struct GqlGraphs {
+    graphs: Vec<IndexedGraph<DynamicGraph>>,
+}
+
+impl GqlGraphs {
```

Comment:
Need to test does this bring in the whole WORKING_DIR or just what is in the cache